packages/web-runtime/src/pages/account.vue

```diff
@@ -152,10 +153,10 @@ export default defineComponent({
     const { $gettext } = language
     const clientService = useClientService()
     const configurationManager = useConfigurationManager()
-    const valuesList = ref()
-    const bundlesList = ref()
-    const selectedLanguageValue = ref()
-    const disableEmailNotificationsValue = ref()
+    const valuesList = ref<SettingValue[]>()
+    const bundlesList = ref<AccountBundle>()
```

Comment:
Why is the bundles list not of type `ref<AccountBundle[]>`? 🤔

Reply:
Because `bundlesList` is actually not a list of bundles but the account bundle. Let my fix that name in a follow-up, just like the new interface names :)

Reply:
thank you 💪